src/app/containers/MediaPlayer/index.jsx

```diff
@@ -57,7 +58,9 @@ const MediaPlayerContainer = ({
   }
 
   const aresMediaBlock = filterForBlockType(blocks, 'aresMedia');
-  const captionBlock = filterForBlockType(blocks, 'caption');
+  const captionBlock =
+    filterForBlockType(blocks, 'caption') ||
+    filterForBlockType(path(['model', 'blocks'], aresMediaBlock), 'caption');
```

Comment:
Does this mean media players are represented with two different block structures in our JSON data?

Reply:
Looks like these two formats relate to their usage in a Media Asset Page or inline in a Story, if that's right a slight readability suggestion:
```javascript
const mapPageCaptionBlock = filterForBlockType(blocks, 'caption');
const storyPageCaptionBlock = filterForBlockType(path(['model', 'blocks'], aresMediaBlock), 'caption');
const captionBlock = mapPageCaptionBlock || storyPageCaptionBlock;
```
Just think this makes it a bit clearer why there are two formats?

Reply:
> Does this mean media players are represented with two different block structures in our JSON data?

It does yes. The structures are slightly different optimo articles vs CPS assets

Reply:
oh okay, so I guess my suggestion still applies but just differentiating between the optimo format and cps format:
```javascript
const optimoCaptionBlock = filterForBlockType(blocks, 'caption');
const cpsCaptionBlock = filterForBlockType(path(['model', 'blocks'], aresMediaBlock), 'caption');
const captionBlock = optimoCaptionBlock || cpsCaptionBlock;
```
Obviously up to you if you think this is actually more readable 🙂